src/qbe_ir_generator.cpp

```diff
@@ -237,7 +258,10 @@ void QbeIrGenerator::Visit(const ReturnStmtNode& ret_stmt) {
   WriteOut_("ret {}\n", FuncScopeTemp{ret_num});
 }
 
-void QbeIrGenerator::Visit(const BreakStmtNode& break_stmt) {}
+void QbeIrGenerator::Visit(const BreakStmtNode& break_stmt) {
+  assert(!label_views_of_jumpable_blocks.empty());
```

Comment:
This should already be checked by TypeChecker whether this break statement is in a loop, am I right? Just to clarify, it's a runtime check for this list?

Reply:
Ha, seems like you implemented it in #92. 😂

Reply:
> This should already be checked by TypeChecker whether this break statement is in a loop, am I right? Just to clarify, it's a runtime check for this list?

Yes, you're correct. The `assert` statement here serves to enforce that the condition should always hold true. If it's violated, it indicates an internal compiler error.